src/main/java/gameframework/drawing/InputManager.java

```diff
@@ -0,0 +1,51 @@
+package gameframework.drawing;
+
+import java.awt.Canvas;
+import java.awt.event.KeyListener;
+import java.awt.event.MouseListener;
+
+/**
+ * Manager that handles all the inputs which must have an effect on a given canvas.
+ */
+public class InputManager {
+	
+	/**
+	 * The canvas whose inputs are managed by this manager
+	 */
+	private final Canvas canvas;
+
+	/**
+	 * Constructor
+	 * @param canvas the canvas whose inputs will be managed
+	 */
+	public InputManager(Canvas canvas) {
+		this.canvas = canvas;
+	}
+	
+	public void addKeyListener(KeyListener keyStr) {
+		canvas.addKeyListener(keyStr);
```

Comment:
I have the impression that this is deprecated. Why do your code uses code that you deprecated?